src/snowflake/snowpark/modin/plugin/_internal/apply_utils.py

```diff
@@ -1356,3 +1363,15 @@ def groupby_apply_sort_method(
             else GroupbyApplySortMethod.GROUP_KEY_APPEARANCE_ORDER
         )
     )
+
+
+def is_supported_snowpark_python_function(func: AggFuncType) -> bool:
+    """Return True if the `func` is a supported Snowpark Python function."""
+    func_module = inspect.getmodule(func)
+    if functions != func_module:
```

Comment:
Doesn't seem like checking the module is necessary if you are checking the function reference as well, but it probably doesn't hurt. Were you concerned about something in particular here?

Reply:
I just use this to report NotImplementedError for those not in the supported list.